### PR TITLE
Upgrade Agora submodule + vibe.d

### PR DESCRIPTION
**Pull request by AndrejMitrovic**
@bpfkorea/core  I can't get this to work properly with the latest `v0.x.x` agora nodes.

Faucet sends 8 transactions and the receiving node accepts them, but then the node fails to nominate anything.

I think v0.x.x is broken..

### CODE REVIEW

**Comment by AndrejMitrovic** [comment]
It looks like the `trace` logger failed to show that a node IP was banned. I'll submit a separate issue for that.

**Comment by AndrejMitrovic** [comment]
Ok faucet does indeed work. I just used the wrong addresses and because of https://github.com/bpfkorea/agora/issues/1540 I failed to realize the nodes failed to connect.

**Comment by kchulj** [comment]
```
Executing command: ["git", "describe"]
Exception happened while trying to determine the version number: object.Exception@source/agora/cli/version/main.d(50): Command failed: ["git", "describe"]
----------------
Please make sure you are building from a git repository or the environment variable AGORA_VERSION is set to a non-empty string.
```
I have seen this locally. Looks like the tags disappeared again somehow.

**Comment by ferencdg** [comment]
easiest fix would be just to append --always as the third element of the array
https://github.com/bpfkorea/agora/blob/336ff7502aa9088028c4b6c6e4f9f870d7e262ee/source/agora/cli/version/main.d#L42 

so it would not require tags anymore.

**Comment by ferencdg** [comment]
you can also just set the AGORA_VERSION env variable as the error message says

**Comment by Geod24** [comment]
Just set it to `HEAD`, as you don't use it anyways.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -26,4 +26,6 @@ jobs:
         sudo apt-get install libsodium-dev
 
     - name: Test compilation
-      run: dub build
+      run: |
+        echo "AGORA_VERSION=HEAD" >> $GITHUB_ENV
+        dub build
```
Comment (Geod24) [inline .github/workflows/main.yml:31]: For some reasons, Github decided that versions exported this way will only be accessible from the *next* step.

Comment (ferencdg) [reply .github/workflows/main.yml:31]: yes, that is in line with the docs:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

how about:
```
AGORA_VERSION=HEAD dub build
```

Comment (Geod24) [reply .github/workflows/main.yml:31]: Yeah this works, and so does:
```yaml
- name: Test compilation
   env:
     - AGORA_VERSION: HEAD
   run: dub build
```

**Review by Geod24 — APPROVED** [review]

**Pull request merged**